src/BuildingRegistry.Projections.Legacy/BuildingSyndication/BuildingSyndicationProjections.cs

```diff
@@ -1138,6 +1138,15 @@ await context
                 }, ct);
             });
 
+            When<Envelope<BuildingUnitRegularizationWasCorrected>>(async (context, message, ct) =>
+            {
+                await context.CreateNewBuildingSyndicationItem(message.Message.BuildingPersistentLocalId, message, item =>
+                {
+                    var unit = item.BuildingUnitsV2.Single(y => y.PersistentLocalId == message.Message.BuildingUnitPersistentLocalId);
+                    unit.HasDeviation = true;
```

Comment:
I think we should add `unit.Version = message.Message.Provenance.Timestamp;` Also include in test if possible.